client/src/components/user/Logout.js

```diff
@@ -0,0 +1,22 @@
+import { Button } from '../common'
+import { useDispatch, useSelector } from 'react-redux'
+import { logout, selectUser } from '../../redux/reducers/userSlice'
+export function Logout() {
+  const user = useSelector(selectUser)
+  const dispatch = useDispatch()
+  const handleLogout = (e) => {
+    e.preventDefault()
+    dispatch(logout())
+  }
+
+  return (
+    <div>
+      <Button
+        className='fixed bottom-2 bg-red-500 bg-opacity-80 text-white hover:bg-red-300'
```

Comment:
If we add `m-2 w-full`, does that make the button span the width of the sidebar?

Reply:
Edit: it's ok

Reply:
that made it span the width of the page for me, also not sure if you ran the client on your end but did were your login/logout buttons grey?